### ISSUE TITLE
Smile overlaps time when strings are long

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/fractions-common#108]
**Test device:**
Dell
**Operating System:**
Win 10
**Browser:**
Chrome
**Problem description:**
For https://github.com/phetsims/QA/issues/367.  Not in published version
When you have the time visible and strings are long, the smile when you get an answer right will cover that time
**Steps to reproduce:**
1. String Test is double
2. Set timer to on
3. Enter a level and get an answer right

**Screenshots:**
![longsmile](https://user-images.githubusercontent.com/41024075/61495110-42e3bd80-a975-11e9-9a09-98984e249e46.png)

Troubleshooting information (do not edit):

<details>

Name: ‪Fraction Matcher‬
URL: https://phet-dev.colorado.edu/html/fraction-matcher/1.2.0-rc.2/phet/fraction-matcher_all_phet.html
Version: 1.2.0-rc.2 2019-07-15 16:22:10 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.142 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jonathanolson** [comment]
@ariel-phet I can add a more restrictive maxWidth, which would result in:

<img width="200" alt="Screen Shot 2019-08-01 at 10 48 24 AM" src="https://user-images.githubusercontent.com/480878/62311872-ffe01a80-b449-11e9-81ab-b3aa07911bf2.png">

But halving the padding between the text nodes might be better:

<img width="209" alt="Screen Shot 2019-08-01 at 10 49 18 AM" src="https://user-images.githubusercontent.com/480878/62311917-1a19f880-b44a-11e9-8131-24f265659a92.png">
<img width="208" alt="Screen Shot 2019-08-01 at 10 49 53 AM" src="https://user-images.githubusercontent.com/480878/62311951-29994180-b44a-11e9-9b60-5c0ec392f476.png">

Is it ok to reduce the padding between text nodes here?

**Comment by ariel-phet** [comment]
@jonathanolson agreed with reducing padding between the text nodes, I think that is a nice change anyhow.

**Comment by jonathanolson** [comment]
Patched in master and RC branches. @KatieWoe can you verify?

**Comment by KatieWoe** [comment]
Looks good in Master
